ext/node/polyfills/internal/fs/handle.ts

```diff
@@ -24,6 +24,12 @@ export class FileHandle extends EventEmitter {
   ): Promise<string | Buffer> {
     return promises.readFile(this, opt);
   }
+
+  close(): Promise<void> {
+    return new Promise((resolve) => {
+      resolve(Deno.close(this.fd));
+    });
```

Comment:
Because Deno.close is synchronous, you can just return a pre-resolved promise.

```suggestion
    // Note that Deno.close is not async
    return Promise.resolve(Deno.close(this.fd))
```